app/mm-redux/reducers/entities/channel_categories.ts

```diff
@@ -3,60 +3,77 @@
 
 import {combineReducers} from 'redux';
 
-import {CategoryTypes} from '../../constants/channel_categories';
-
-import {TeamTypes} from '@mm-redux/action_types';
+import {ChannelCategoryTypes, TeamTypes, UserTypes, ChannelTypes} from '@mm-redux/action_types';
 
 import {GenericAction} from '@mm-redux/types/actions';
 import {ChannelCategory} from '@mm-redux/types/channel_categories';
-import {Team, TeamMembership} from '@mm-redux/types/teams';
+import {Team} from '@mm-redux/types/teams';
 import {$ID, IDMappedObjects, RelationOneToOne} from '@mm-redux/types/utilities';
+import {removeItem} from '@mm-redux/utils/channel_utils';
 
 export function byId(state: IDMappedObjects<ChannelCategory> = {}, action: GenericAction) {
     switch (action.type) {
-    case TeamTypes.RECEIVED_MY_TEAM_MEMBER: {
-        // This will be removed once categories are sent by the server
-        const member: TeamMembership = action.data;
+    case ChannelCategoryTypes.RECEIVED_CATEGORIES: {
```

Comment:
is this a direct port from the webapp? can you provide a doc for this to be handled in Gekidou? same questions for the selectors

Reply:
Yup will add comments.